### PR TITLE
More tutorials

### PR DESCRIPTION
**Pull request by koaning**
Adding code for some of the Prodigy shorts. Also updated the main README to notify users of this.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -105,6 +105,17 @@ You can edit the code in the recipe script to customize how Prodigy behaves.
...
 | [`records.link`](contrib/dedupe) | @kabirkhan | Link records across multiple datasets using the [`dedupe`](https://github.com/dedupeio/dedupe) library. |
 
+### Tutorial recipes
+
+These recipes have made an appearance in one of our tutorials. 
+
+| Recipe                                                      | Description                                                                                   |
+| ----------------------------------------------------------- | --------------------------------------------------------------------------------------------- |
+| [`span-and-textcat`](tutorials/span-and-textcat/)           | Do both spancat and textcat annotations at the same time. Great for chatbots!                 |
+| [`terms.from-ner`](tutorials/terms-from-ner/)               | Generate terms from previous NER annotations                                                  |
```
Comment (svlandeg) [inline README.md]:
```suggestion
| [`terms.from-ner`](tutorials/terms-from-ner/)               | Generate terms from previous NER annotations.                                                 |
```

File: tutorials/audio-with-transcript/recipe.py [hunk 2]
```diff
@@ -0,0 +1,72 @@
+import prodigy 
+from typing import List, Optional, Union, Iterable
+
```
Comment (svlandeg) [inline tutorials/audio-with-transcript/recipe.py]:
```suggestion
from typing import List, Optional, Union, Iterable
import prodigy 
```

**Review by svlandeg — COMMENTED** [review]
I only gave this a quick look-over, as perhaps the actual code of the recipes has already been reviewed on other occassions?

Either way, definitely a good idea to add these to the repo.

**Comment by koaning** [comment]
The code has all appeared in a Prodigy (short) video before, so I also don't think an in-depth PR is needed. Also; this PR is mainly to make sure most of our content can be found here so people are also inspired to have a look at other recipes.

**Comment by koaning** [comment]
I added your suggestions, @svlandeg shall I merge?

**Pull request merged**